hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/metrics/RpcMetrics.java

```diff
@@ -147,6 +147,8 @@ public static RpcMetrics create(Server server, Configuration conf) {
   MutableCounterLong rpcRequeueCalls;
   @Metric("Number of successful RPC calls")
   MutableCounterLong rpcCallSuccesses;
+  @Metric("Number of observer namenode rejected RPC calls")
+  MutableCounterLong rpcCallsRejectedByObserver;
```

Comment:
Thanks @haiyang1987  for your report. And thanks @simbadzina @goiri for your review.

I'm just concern if it's reasonable that `rpcCallsRejectedByObserver` is in `RpcMetrics.java`.  I think `rpcCallsRejectedByObserver` is a logic of namenode, so how about changing this logic as a HDFS issue, not common issue?

Reply:
Thanks @ZanderXu for your comment.
@goiri @simbadzina What do you think? If necessary, I will modify it into an HDFS issue to achieve this requirement.
thanks~